packages/flutterfire_ui/lib/src/firestore/table_builder.dart

```diff
@@ -819,8 +846,9 @@ class _Source extends DataTableSource {
 
     final doc = _previousSnapshot!.docs[index];
     final data = doc.data();
-
-    return DataRow(
+    final colList = getHeaders().keys.toList();
+    return DataRow.byIndex(
```

Comment:
What is this change about?

Reply:
byIndex, will assign a unique index per row, so sorting will work as expected when implemented, and this is anyway a good practice to unique key per DataRow for helping the rendering behind the scene.